src/components/DashboardResultIndicatorsSection.vue

```diff
@@ -605,5 +613,9 @@ export default {
     padding: 0.3rem;
     font-size: 12px;
   }
+  &__target {
+    color: var(--color-grey-600);
+    font-size: 12px;
```

Comment:
Skal vi bruke verdiene fra `src/style/typography.scss` her?
```scss
@use '@/styles/typography';
```
Blir vel i så fall `font-size: typography.$font-size-0;` for `.progressTarget__target` og `.progressTarget__progress`. Kan kanskje samtidig bruke `typography.$font-size-1` for `.progressTarget__title` også? :)

Reply:
Vet ikke helt om Figma-skissene er oppdatert siden sist (sorry hvis ikke), men samtidig legge på `font-weight: 500;` for `.progressTarget__title`, og sette `font-size: typography.$font-size-4;` for `.progressTarget__value`? :innocent:

Reply:
Takk for gode innspill, skal se på det asap :)